go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/veqryn/slog-json
 
-go 1.21
+go 1.24
```

Comment:
Do you know if this is a requirement, or if the library will work with an older version?

Reply:
Dear @veqryn I've changed it to 1.24 because the original lib is also using 1.24
https://github.com/go-json-experiment/json/blob/master/go.mod
IMHO any lower version wouldn't make sense here.